packs/excel/actions/get_keys.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: "get_keys"
+runner_type: "run-python"
+description: "returns an array of the keys for a given sheet in the excel file"
+enabled: true
+entry_point: "get_keys.py"
```

Comment:
I tested it locally and it looks like this action returns first column from each of the rows.

I think it would also make sense to allow user to retrieve all the columns in the first row as well - usually first line is the header which contains all the keys. Or perhaps we should add new action (e.g. `get_keys_for_rows`) and rename this one `get_keys_for_columns` or similar?